molgenis-data-vcf/src/main/java/org/molgenis/data/vcf/VcfRepository.java

```diff
@@ -177,6 +186,15 @@ public String apply(Allele allele)
 			{
 				val = null;
 			}
+			// if a flag field exists in the line, than this field is true, although the value is null
```

Comment:
In this part of the function we have an vcfInfo in our hands instead of a vcfMetaInfo. So additional code would be needed to find the correct metainfo to check for flag...